scripts/run_pa11y.js

```diff
@@ -0,0 +1,92 @@
+const spawn = require('cross-spawn');
```

Comment:
@tgardnerFusion I'm curious of your thoughts on this script. On the plus side, it automates the pa11y process which normally has to be run manually against a running server. On the negative side, it is a cumbersome script that may scare devs Thoughts?